src/test/util_string_tests.cpp

```diff
@@ -83,4 +86,15 @@ BOOST_AUTO_TEST_CASE(ConstevalFormatString_NumSpec)
     FailFmtWithError<1>("%1$", err_term);
 }
 
```

Comment:
We could add the examples from `bitcoin-cli` here:
```C++
BOOST_AUTO_TEST_CASE(ConstevalFormatString_SpecificUsageTests)
{
    // These format strings contain '*' and should skip validation
    PassFmt<6>("<->   type   net  v  mping   ping send recv  txn  blk  hb %*s%*s%*s ");
    PassFmt<5>("%*s %-*s%s\n");
    PassFmt<22>(
        "%3s %6s %5s %2s%7s%7s%5s%5s%5s%5s  %2s %*s%*s%*s%*i %*s %-*s%s\n");
    PassFmt<2>("                        ms     ms  sec  sec  min  min                %*s\n\n");
    PassFmt<4>("\n%-*s    port %6i    score %6i");
    PassFmt<3>("%*s %s\n");
}
```

Reply:
Addressed in #30999, resolving.